valeriyapetrova/hometask-5/4.1.sql

```diff
@@ -0,0 +1 @@
+SELECT * FROM actor WHERE last_name IN ('Zellweger');
```

Comment:
`... last_name IN ('Zellweger')` is fine but it also may be replaced with `last_name = 'Zellweger'`